### PR TITLE
Added rpc_shutdown documentation.

### LINKED ISSUE #734: Add documentation to the rpc_shutdown add-on
The [`rpc_shutdown`](https://github.com/home-assistant/hassio-addons/tree/master/rpc_shutdown) add-on, currently, does not have documentation in its repository.
We should add it, so the users see the documentation and information in the Hass.io add-on and add-on store pages in the Home Assistant frontend.

A `README.md` file needs to be added. The documentation of the [`check_config`](https://github.com/home-assistant/hassio-addons/blob/master/check_config/README.md) add-on is a good example that can be used.

### PR DESCRIPTION
**Pull request by RahulRavishankar**
Closes #734

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @RahulRavishankar, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;hassio-addons#746).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: rpc_shutdown/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    },
+    {
+      "alias": "test-pc-2",
+      "address": "192.168.0.2",
+      "credentials": "user%password"
+    }
+  ]
+}
+```
+
```
Comment (frenck) [inline rpc_shutdown/README.md]: Could you please document the individual options below here? To tell people what they are and what to expect?

File: rpc_shutdown/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+## Home Assistant Configuration
+
```
Comment (frenck) [inline rpc_shutdown/README.md]: Please add a bit of text/introduction to what this is.

File: rpc_shutdown/README.md (hunk 1 above)
Comment (RahulRavishankar) [reply rpc_shutdown/README.md]: Yes!

File: rpc_shutdown/README.md (hunk 2 above)
Comment (RahulRavishankar) [reply rpc_shutdown/README.md]: I'll do that

Comment (RahulRavishankar) [reply rpc_shutdown/README.md]: I've made the changes requested. Please do review them.

**Comment by frenck** [comment]
Thanks, @RahulRavishankar!

I've adjusted the docs a big with some small things and made the layout/styling similar to the other README's we already have.

Happy Hacktoberfest 🎉

**Pull request merged**